service/initialise.go

```diff
@@ -37,7 +37,8 @@ var GetResponder = func() Responder {
 var GetCantabularClient = func(cfg *config.Config) CantabularClient {
 	return cantabular.NewClient(
 		cantabular.Config{
-			Host: cfg.CantabularURL,
+			Host:       cfg.CantabularURL,
+			ExtApiHost: cfg.CantabularExtURL,
```

Comment:
most places seem to configure timeout at this point too. not here?

Reply:
I think it's not used if we provide the http client.
The following line (`dphttp.NewClient()`) will provide the client configured with the default paramters, defiend in dp-net library: https://github.com/ONSdigital/dp-net/blob/main/http/client.go#L42